python/OnnxUdf/onnx_udf.py

```diff
@@ -0,0 +1,22 @@
+import sys
+from openeo.udf import XarrayDataCube
+from typing import Dict
+import xarray as xr
+from openeo.udf.debug import inspect
+
+
+def apply_datacube(cube: XarrayDataCube, context: Dict) -> XarrayDataCube:
+    sys.path.insert(0, "tmp/extra_venv")
+    import onnxruntime as ort
```

Comment:
I don't think there is a reason to do this onnx import stuff inside the function. This can just be toplevel, right?